style.css

```diff
@@ -14,12 +19,12 @@ body {
 }
 
 #padding-ctn {
-  overflow-x: hidden;
-  overflow-y: auto;
   display: flex;
   flex-direction: column;
   justify-content: center;
   padding: 0 3rem;
+  padding-bottom: 3rem;
+  gap: 3rem;
```

Comment:
- [ ] Please remove extra margin/padding from your `skills` cards (as they seem to be squashed from the sides) to match with the provided [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=SQaxNf7Wr55PlmeV-0)

|Figma design| Your design|
|----|----|
|<img width="621" alt="Screenshot 2023-06-05 at 10 51 30 PM" src="https://github.com/Movgto/my-portfolio/assets/44798044/4efe7bfe-0c37-4ee2-ae20-4cbac498bb25">|<img width="368" alt="Screenshot 2023-06-05 at 10 52 07 PM" src="https://github.com/Movgto/my-portfolio/assets/44798044/1b7a6553-2c3d-42c5-9a8e-009637032baf">|